### DISCUSSION TITLE
Not showing most recent posts; and a configuration question

### DISCUSSION THREAD

**Discussion by amca01** [chipzoller/hugo-clarity#123] [Q&A]
Hello,

I thought I'd ask two questions at once:

1.  For some reason, when I run `hugo server` on my local site it doesn't show the most recent blog posts.  They are there, under their tags, but they don't appear at the top of the posts list - they should be the first things seen on the blog's home page.   Neither do they appear under "Home" or "All posts".  The only way to see them is by using their tag.  How can I have them show on the main screen?
2.  This might be a standard Hugo question - but how can I set options for KaTeX?   At the KaTeX site, there is [documentation for configuration](https://katex.org/docs/options.html), but I can't see how to add it either to the `math.html` file in the `layouts/partial` directory ... or anywhere else.  

Anyway - thanks.  I should also say I very much like the look and feel of Clarity, in many ways cleaner and more elegant than the previous theme I'd been using (Blackburn) which I had to drop as it hasn't been upgraded in about three years.  

cheers,
Alasdair

**Comment by chipzoller** [comment]
Jeez, so sorry to have missed this discussion even after so long. Apologies for that. Are you all set, Alasdair?
